script/releases/v0.5.1-rewardsv2/3-multisig.s.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import {MultisigCall, MultisigCallUtils, MultisigBuilder} from "zeus-templates/templates/MultisigBuilder.sol";
+import {SafeTx, SafeTxUtils} from "zeus-templates/utils/SafeTxUtils.sol";
+import {Queue} from "./2-multisig.s.sol";
+import {EigenLabsUpgrade} from "../EigenLabsUpgrade.s.sol";
+import {TimelockController} from "@openzeppelin/contracts/governance/TimelockController.sol";
+import {RewardsCoordinator} from "src/contracts/core/RewardsCoordinator.sol";
+import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+
+contract Execute is Queue {
+    using MultisigCallUtils for MultisigCall[];
+    using SafeTxUtils for SafeTx;
+    using EigenLabsUpgrade for *;
+
+    /**
+     * @dev Overrides the previous _execute function to execute the queued transactions.
+     */
+    function _runAsMultisig() internal override {
+        bytes memory call = _getMultisigTransactionCalldata();
+        TimelockController timelock = TimelockController(payable(this._timelock()));
+        timelock.execute(
+            this._executorMultisig(),
+            0,
+            call,
+            0,
+            bytes32(0)
+        );
+    }
+
+    function testDeploy() override public {}
+
+    function testExecute() public {
+        zUpdateUint16(string("REWARDS_COORDINATOR_DEFAULT_OPERATOR_SPLIT_BIPS"), uint16(1000)); // mocking step 2.
+
+        // 1. run the queue script.
+        vm.startPrank(this._operationsMultisig());
+        super._runAsMultisig();
+        vm.stopPrank();
+
+        TimelockController timelock = this._timelock();
+
+        // 2. run the execute script above.
+        bytes memory call = _getMultisigTransactionCalldata();
+        bytes32 txHash = timelock.hashOperation(this._executorMultisig(), 0, call, 0, 0);
+        
+        assertEq(timelock.isOperationPending(txHash), true, "Transaction should be queued and pending.");
+
+        vm.warp(block.timestamp + timelock.getMinDelay()); // 1 tick after ETA.
+
+        assertEq(timelock.isOperationReady(txHash), true, "Transaction should be executable.");
+        
+        zSetMultisigContext(this._protocolCouncilMultisig());
```

Comment:
Is Zeus supposed to handle the queuing up of the Protocol Council Multisig tx too or is that done independently in the Safe UI by the protocol council?

Reply:
a protocol council multisig member will need to run zeus to produce the tx